ProcessMaker/screen-builder
# Fix value of calculated inside calculated

## Issue & Reproduction Steps
Unable to get the result from a multiplication on Calculated property using a field with default value as 0

Expected behavior: 
For the example, value of `total` should be equal to `b`

Actual behavior: 
For the example, the value of `total` is undefined and `b` is `0`

## Solution
- Fix the computed property update

## How to Test
- Import the screen
- Click on Preview
- Check the variable is loaded

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-6523

## Code Review Checklist
